.github/workflows/validation.yml

```diff
@@ -0,0 +1,43 @@
+name: Validate modified chains
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
```

Comment:
This trigger type allows github secrets (such as our rpc endpoints) to be shared with forked prs. Github documentation [here](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target).